iso.c

```diff
@@ -26,13 +26,14 @@ extern RD_BOOL g_encryption_initial;
 extern RDP_VERSION g_rdp_version;
 extern RD_BOOL g_use_password_as_pin;
 
-static RD_BOOL g_negotiate_rdp_protocol = True;
-
 extern char *g_sc_csp_name;
 extern char *g_sc_reader_name;
 extern char *g_sc_card_name;
 extern char *g_sc_container_name;
 
+// MultiMonitors : external declaration - see xwin.c for variables
+extern RD_BOOL g_monitors_supported;
```

Comment:
As this variable is only dependable upon EXTENDED_CLIENT_DATA_SUPPORTED flag maybe we should call
it **"g_extended_data_supported"** instead?

Reply:
yes, good suggestion